easybuild/easyconfigs/o/OpenMPI/OpenMPI-1.6.3-iccifort-2013.3.163.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'ConfigureMake'
+
+name = 'OpenMPI'
+version = '1.6.3'
+
+homepage = 'http://www.open-mpi.org/'
+description = """The Open MPI Project is an open source MPI-2 implementation."""
+
+toolchain = {'name': 'iccifort', 'version': '2013.3.163'}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://www.open-mpi.org/software/ompi/v%(version_major_minor)s/downloads']
+
+configopts = '--with-threads=posix --enable-shared --enable-mpi-thread-multiple --with-openib '
+configopts += '--enable-mpirun-prefix-by-default '  # suppress failure modes in relation to mpirun path
+configopts += '--with-hwloc=$EBROOTHWLOC --with-pmi '  # hwloc support
```

Comment:
@boegel: We added the --with-pmi option because we needed it to be able to launch the srun command in SLURM:
http://slurm.schedmd.com/mpi_guide.html#open_mpi

I can remove the option to be consistent with the other easyconfigs for OpenMPI and we can just have it locally, in our modified configs.